.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -3,6 +3,13 @@ on:
   schedule:
     - cron: '*/5 * * * *'
   workflow_dispatch:
+  workflow_run:
+    workflows: ["Unit Tests - Latest Dependencies"]
+    branches:    
+      - 'latest-dep-update-[a-f0-9]+'
```

Comment:
Need a double check on this syntax
- https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
```
dep_update = '^latest-dep-update-[a-f0-9]{7}$'
min_dep_update = '^min-dep-update-[a-f0-9]{7}$'
```

Reply:
Not sure if the github syntax supports `[a-f0-9]{7}` but we could put `[a-f0-9]` 7 times in a row . . .

Reply:
That's why I put `+`, which means `+: Matches one or more of the preceding character.`.